packages/kbn-coloring/src/shared_components/color_mapping/categorical_color_mapping.tsx

```diff
@@ -15,22 +15,26 @@ import { Container } from './components/container/container';
 import { ColorMapping } from './config';
```

Comment:
I observed this small bug when switching between categorical/numerical color palettes. 
1. Create a datatable with a last value dimension and categorical color palette set.
2. switch to a numerical operation, like 'sum'
3. Click on the color palette - the palette input is not filled properly. I'll pass you the video in a PM (github limits won't allow me to do so)

Reply:
Good catch, fixed this in 77cd4094d32b80394bcbef984e927b9840257816